server/controllers/Project.js

```diff
@@ -25,6 +25,35 @@ exports.projectById = (req, res, next, id) => {
         })
 }
 
+exports.getProjects = (req, res) => {
+    const { _id } = req.profile;
+    const order = req.query.order || 'asc';
+    const sortBy = req.query.sortBy || 'fundingDeadline';
+    const limit = parseInt(req.query.limit) || 6;
+    const cutoff = new Date();
+
+    const filterOptions = {
+        user: { $ne: _id },
```

Comment:
Just curious is this what filters out the id? wasn't sure what $ne meant

Reply:
yeah, it means not equal to: 
https://docs.mongodb.com/manual/reference/operator/query/ne/